src/modules/design/design.html

```diff
@@ -140,5 +140,17 @@
       </template>
     </modal>
 
+    <modal show.bind="diagramDetail.showDiagramExistingModal"
+          header-text="Warning: Diagram already exists!">
+      <template replace-part="modal-body">
+        The target solution already contains a diagram with that name and ID.
```

Comment:
Please add a line break.

```suggestion
        The target solution already contains a diagram with that name and ID.<br>
```